python/lsst/verify/bin/jobReporter.py

```diff
@@ -112,8 +120,7 @@ def run(self):
 
```

Comment:
I can't comment on the line above but you do not need to create a list from the return value of queryDatasets if you are immediately going to loop over it. It's an iterator itself so the list is overhead you don't want.